src/main/steps/noResponseJourneySequence.ts

```diff
@@ -57,4 +76,90 @@ export const noResponseJourneySequence: Step[] = [
       return data.applicant2AddressOverseas === YesOrNo.YES ? NO_NEW_ADDRESS : SERVE_AGAIN;
     },
   },
+  {
+    url: NO_NEW_ADDRESS,
+    getNextStep: (data: Partial<CaseWithId>): PageLink => {
+      switch (data.applicant1NoResponseNoNewAddressDetails) {
+        case NoResponseNoNewAddressDetails.IN_PERSON_SERVICE: {
+          return PARTNER_IN_PERSON;
+        }
+        case NoResponseNoNewAddressDetails.ALTERNATIVE_SERVICE: {
+          return ALTERNATIVE_SERVICE_APPLICATION;
+        }
+        case NoResponseNoNewAddressDetails.NO_CONTACT_DETAILS: {
+          return OWN_SEARCHES;
+        }
+        default: {
+          return HUB_PAGE;
+        }
+      }
+    },
+  },
+  {
+    url: PARTNER_IN_PERSON,
+    getNextStep: (data: Partial<CaseWithId>): PageLink => {
+      switch (data.applicant1NoResponseProcessServerOrBailiff) {
+        case NoResponseProcessServerOrBailiff.PROCESS_SERVER: {
+          return PROCESS_SERVER;
+        }
+        case NoResponseProcessServerOrBailiff.COURT_BAILIFF: {
+          return BAILIFF_SERVICE_APPLICATION;
+        }
+        default: {
+          return HUB_PAGE;
+        }
```

Comment:
We don't use this approach of defaulting to a hub page elsewhere in the application steps, as the nextStep function only calls the next step if the form on a page is complete. Wondering if it will be used in live and whether redirecting to the hub would make sense to the user, or we could simplify:
```suggestion
      return data.applicant1NoResponseProcessServerOrBailiff === NoResponseProcessServerOrBailiff.PROCESS_SERVER
        ? PROCESS_SERVER;
        :  BAILIFF_SERVICE_APPLICATION;
```